### PR TITLE
sozluk-content/terms for Devops, Cloud, Http-Methods 

### LINKED ISSUE #525: (sozluk-content/terms): Writing descriptions of empty .mdx files.
_The list will be updated as additions are made._

- [ ] [client-server-communication.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/client-server-communication.mdx)
- [x] [cloud.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/cloud.mdx) @akselsaatci 
- [ ] [content-caching.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/content-caching.mdx)
- [ ] [creational-pattern.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/creational-pattern.mdx)
- [ ] [database-operations.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/database-operations.mdx)
- [ ] [database.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/database.mdx)
- [ ] [decision-support-system.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/decision-support-system.mdx)
- [x] [devops.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/devops.mdx) @akselsaatci 
- [ ] [discord-bot.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/discord-bot.mdx)
- [ ] [function-as-a-service.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/function-as-a-service.mdx)
- [ ] [geographic-distribution.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/geographic-distribution.mdx)
- [ ] [http-access-control.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/http-access-control.mdx)
- [x] [http-methods.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/http-methods.mdx) @akselsaatci 
- [ ] [lean-startup.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/lean-startup.mdx)
- [ ] [load-sharing.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/load-sharing.mdx)
- [ ] [machine-learning.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/machine-learning.mdx)
- [ ] [metric.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/metric.mdx)
- [ ] [product-development.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/product-development.mdx)
- [ ] [product-management.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/product-management.mdx)
- [ ] [prototyping.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/prototyping.mdx)
- [ ] [rate-control.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/rate-control.mdx)
- [ ] [reliability.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/reliability.mdx)
- [ ] [resource-allocation.mdx](https://github.com/kamp-us/monorepo/blob/dev/packages/sozluk-content/terms/resource-allocation.mdx)

### PR DESCRIPTION
**Pull request by akselsaatci**
# Description

Please describe your changes. Be descriptive enough to reduce churn for review process.

### Checklist

- [x] discord username: `akselsaatci`
- [x] Part of #525 (devops,cloud,http-methods)
- [x] PR must be created for an issue from issues under "In progress" column from [our project board](https://github.com/orgs/kamp-us/projects/2/views/1).
- [x] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules, and should include the app/package/service name in the title. For example, a title like "docs(@kampus-apps/pano): Add README.md" can be used.
- [x] Related file selection: Only relevant files should be touched and no other files should be affected.
- [x] I ran `npx turbo run` at the root of the repository, and build was successful.
- [x] I installed the npm packages using `npm install --save-exact <package>` so my package is pinned to a specific npm version. Leave empty if no package was installed. Leave empty if no package was installed with this PR.

### How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

### Sources
https://medium.com/devopsturkiye/devops-nedi%CC%87r-9f7e39b63b9c
https://aws.amazon.com/what-is-cloud-computing/
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
and of course https://chat.openai.com/

### CODE REVIEW

File: packages/sozluk-content/terms/cloud.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Cloud
+tags:
+  - cloud computing
+  - cloud services
```
Comment (igdiaysu) [inline packages/sozluk-content/terms/cloud.mdx]:
```suggestion
  - cloud-computing
  - cloud-services
```

File: packages/sozluk-content/terms/devops.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Devops
+tags:
+  - Continuous Integration
+  - Continuous Deployment
+  - Deployment
+  - Pipeline
```
Comment (igdiaysu) [inline packages/sozluk-content/terms/devops.mdx]:
```suggestion
  - continuous-integration
  - continuous-deployment
  - deployment
  - pipeline
```

File: packages/sozluk-content/terms/http-methods.mdx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: HTTP Methods
+tags:
+  - HTTP
+  - GET POST PUT DELETE PATCH HEAD OPTIONS CONNECT TRACE
```
Comment (igdiaysu) [inline packages/sozluk-content/terms/http-methods.mdx]:
```suggestion
  - http
  - get
  - post
  - put
  - delete
  - patch
  - head
  - options
  - connect
  - trace
```

File: packages/sozluk-content/terms/http-methods.mdx [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+---
+
+Http metotları sunucu ile istemci arasında iletişim kurmak için kullanılan komutlardır. Http protokolü 9 adet metot tanımlamıştır. Bunlar : GET, POST, PUT, DELETE, PATCH, HEAD, OPTIONS, CONNECT ve TRACE'dir.
```
Comment (igdiaysu) [inline packages/sozluk-content/terms/http-methods.mdx]:
```suggestion
HTTP metotları sunucu ile istemci arasında iletişim kurmak için kullanılan komutlardır. HTTP protokolü 9 adet metot tanımlamıştır: GET, POST, PUT, DELETE, PATCH, HEAD, OPTIONS, CONNECT ve TRACE.  
```

File: packages/sozluk-content/terms/cloud.mdx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+title: Cloud
+tags:
+  - cloud-computing
+  - cloud-services
+---
+
+Bulut bilişimleri fiziksel sunucuları satın almak, sahip olmak ve bakımını yapmak yerine bunları bir sağlayıcı tarafından ulaşmamızı sağlar. Kullanıma göre fiyatlandırılır. İhtiyaçlara göre ölçeklendirilebilir.
```
Comment (aecepoglu) [inline packages/sozluk-content/terms/cloud.mdx]: bunlari -> bunlara ?

**Review by aecepoglu — CHANGES_REQUESTED** [review]
Bir konumda yonelim eki hatali diye dusundum.

Bu dedigim ise degisim istegi degil, sadece dusuncemi paylasiyorum: Sozlukteki cumleler icin kelime sayisi ust limiti getirilmeli. cloud ve devops'un tanimlarini okurken cumlenin basini unuttum.

**Comment by akselsaatci** [comment]
> Bir konumda yonelim eki hatali diye dusundum.
> 
> Bu dedigim ise degisim istegi degil, sadece dusuncemi paylasiyorum: Sozlukteki cumleler icin kelime sayisi ust limiti getirilmeli. cloud ve devops'un tanimlarini okurken cumlenin basini unuttum.

2. dediğinizde haklı olabilirsiniz kısaltmaya çalışacağım

**Comment by Ketcap** [comment]
@akselsaatci @igdiaysu Is there anything else needs to be resolved here ? If not we can just merge this.

**Comment by igdiaysu** [comment]
> @akselsaatci @igdiaysu Is there anything else needs to be resolved here ? If not we can just merge this.

I think it can be merged . But i was not sure because of the above [dialog](https://github.com/kamp-us/monorepo/pull/538#issuecomment-1646932086)

**Review by aecepoglu — APPROVED** [review]

**Review by Ketcap — APPROVED** [review]

**Pull request merged**